.github/workflows/ios-build-and-publish.yml

```
@@ -33,3 +33,22 @@ jobs:
         AWS_REGION: 'us-east-1'
         SOURCE_DIR: 'live/ios-config'
         DEST_DIR: 'remotemessaging/config/staging'
+    - name: Extract Version from JSON
+      id: extract_version
+      run: |
+        VERSION=$(jq -r '.version' live/ios-config/ios-config.json)
+        echo "Extracted Version: $VERSION"
+        echo "VERSION=$VERSION" >> $GITHUB_ENV
+    - name: Trigger Network Request
+      id: trigger_request
+      run: |
+        RESPONSE=$(curl -s -o response.txt -w "%{http_code}" "https://improving.duckduckgo.com/t/m_new_rmf_config_ios?version=${VERSION}")
```

To limit curl output even more you can use the following:
```suggestion
        RESPONSE=$(curl -fLSs -o /dev/null -w "%{http_code}" "https://improving.duckduckgo.com/t/m_new_rmf_config_ios?version=${VERSION}")
```